### PR TITLE
Prepping for next Ariadne release for php7.2

### PR DESCRIPTION
**Pull request by ylebre**

### CODE REVIEW

File: lib/modules/mod_tar.phtml [hunk 1]
```diff
@@ -63,7 +63,7 @@
 				}
 			}
 			$result = $archive->addModify($filelist, "", $srcdir);
-			$archive->_Archive_Tar();
+			$archive->__destruct();
```
Comment (mjrider) [inline lib/modules/mod_tar.phtml]: isn't `unset($achive)` a cleaner way to do this ?

File: lib/stores/store.php [hunk 2]
```diff
@@ -430,9 +434,9 @@ protected function unserialize($value, $path) {
 			}
 
 			if ($decryptedValue[0] === "O" && $decryptedValue[1] === ":") {
-				return unserialize($decryptedValue);
+				return unserialize(self::fixObjectClass($decryptedValue));
 			} else {
-				$dummy = unserialize('O:6:"object":7:{s:5:"value";s:0:"";s:3:"nls";O:6:"object":2:{s:7:"default";s:2:"nl";s:4:"list";a:1:{s:2:"nl";s:10:"Nederlands";}}s:2:"nl";O:6:"object":1:{s:4:"name";s:14:"Crypted object";}s:6:"config";O:6:"object":2:{s:10:"owner_name";s:6:"Nobody";s:5:"owner";s:6:"nobody";}s:5:"mtime";i:0;s:5:"ctime";i:0;s:5:"muser";s:6:"nobody";}');
+				$dummy = self::fixObjectClass(unserialize('O:6:"object":7:{s:5:"value";s:0:"";s:3:"nls";O:6:"object":2:{s:7:"default";s:2:"nl";s:4:"list";a:1:{s:2:"nl";s:10:"Nederlands";}}s:2:"nl";O:6:"object":1:{s:4:"name";s:14:"Crypted object";}s:6:"config";O:6:"object":2:{s:10:"owner_name";s:6:"Nobody";s:5:"owner";s:6:"nobody";}s:5:"mtime";i:0;s:5:"ctime";i:0;s:5:"muser";s:6:"nobody";}'));
```
Comment (mjrider) [inline lib/stores/store.php]: Wouldn't it be better to just fix the serialized string here?

File: lib/modules/mod_tar.phtml (hunk 1 above)
Comment (ylebre) [reply lib/modules/mod_tar.phtml]: agreed and fixed.

File: lib/stores/store.php (hunk 2 above)
Comment (ylebre) [reply lib/stores/store.php]: agreed and fixed.

**Pull request merged**